<launch>

    <arg name="launch_prefix" default="" /> <!--set to value="gdbserver localhost:10000" for remote debugging-->
    <arg name="node_namespace" default="apriltag_ros_multi_image_client_node" /> 
    <arg name="camera_name" default="/eye" />  
    <arg name="image_topic" default="image_rect_color" />
    <!-- Set parameters -->
    <rosparam command="load" file="$(find apriltag_ros)/config/settings.yaml" ns="$(arg node_namespace)" />
    <rosparam command="load" file="$(find apriltag_ros)/config/tags.yaml" ns="$(arg node_namespace)" />
    <rosparam command="load" file="$(find my_hand_eye)/param/move.yaml" ns="$(arg node_namespace)" />

    <node pkg="apriltag_ros" type="apriltag_ros_multi_image_client_node" name="$(arg node_namespace)" clear_params="true" output="screen" launch-prefix="$(arg launch_prefix)" args="/dev/ft_servo">
      <!-- Remap topics from those used in code to those on the ROS network -->
      <remap from="image_rect" to="$(arg camera_name)/$(arg image_topic)" />
      <!-- Camera intrinsic parameters -->
      <param name="fx" type="double" value="788.709302" />
      <param name="fy" type="double" value="940.728627" />
      <param name="cx" type="double" value="932.106780" />
      <param name="cy" type="double" value="578.390364" />
  
    </node>
  
  </launch>
  